index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Indie Games Summit</title>
+    <link rel="stylesheet" href="css/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <main>
+      <nav class="shadow fixed-top navbar navbar-expand-md bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand ps-5 d-none d-md-block" href="#"
+            ><img src="images/CCGSlogo.png" alt=""
+          /></a>
+          <button
+            class="navbar-toggler collapsed"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNavAltMarkup"
+            aria-controls="navbarNavAltMarkup"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="toggler-icon top-bar"> </span>
+            <span class="toggler-icon mid-bar"> </span>
+            <span class="toggler-icon bot-bar"> </span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav ms-auto">
+              <a
+                class="nav-link px-md-3 px-2 active"
+                aria-current="page"
+                href="#"
+                >Home</a
+              >
+              <a class="nav-link px-md-3 px-2" href="about.html">About</a>
+              <a class="nav-link px-md-3 px-2" href="#">Program</a>
+              <a class="nav-link px-md-3 px-2" href="#">Join</a>
+              <a class="nav-link px-md-3 px-2" href="#">Sponsor</a>
```

Comment:
- [ ] Kindly add the top short bar section with social media links, logout, and language links aligned to the right. To match the original layout.

![image](https://user-images.githubusercontent.com/35535130/181620631-d6cd8a0b-d5b5-479e-966f-2094b5ca688d.png)